servers.json

```diff
@@ -739,6 +739,13 @@
         "addresses": [
             "mc.lynxmc.ga"
         ]
+    },
+	  {
```

Comment:
This line contains a tab character that needs to be converted to spaces

Reply:
Thank you, I fixed it.

Reply:
Thank you, now all issues must have been be solved.